autogen/agentchat/contrib/gpt_assistant_agent.py

```diff
@@ -79,7 +84,7 @@ def __init__(
                     name=name,
                     instructions=instructions,
                     tools=llm_config.get("tools", []),
-                    model=llm_config.get("model", "gpt-4-1106-preview"),
+                    model=model_name,
                     file_ids=llm_config.get("file_ids", []),
```

Comment:
A cleaner way is to make everything about `create` handled by `OpenAIWrapper` and not here.
It's OK to do that in a separate PR.